okio/src/jvmMain/kotlin/okio/JvmOkio.kt

```diff
@@ -204,6 +206,26 @@ fun Sink.cipherSink(cipher: Cipher): CipherSink = CipherSink(this.buffer(), ciph
  */
 fun Source.cipherSource(cipher: Cipher): CipherSource = CipherSource(this.buffer(), cipher)
 
+/**
+ * Returns a sink that uses [mac] to hash [this].
+ */
+fun Sink.macSink(mac: Mac): HashingSink = HashingSink(this, mac)
+
+/**
+ * Returns a source that uses [mac] to hash [this].
+ */
+fun Source.macSource(mac: Mac): HashingSource = HashingSource(this, mac)
+
+/**
+ * Returns a sink that uses [digest] to hash [this].
+ */
+fun Sink.digestSink(digest: MessageDigest): HashingSink = HashingSink(this, digest)
```

Comment:
rename to `hashingSink` ? I don’t think `digest` is as guessable as hashingSink for autocomplete!